web-registry/src/lib/transform.ts

```diff
@@ -117,7 +117,7 @@ export function buildIssuanceModalData(
         creditVintage.endDate,
       ),
       monitoringPeriods,
-      projectName: project.metadata?.['http://schema.org/name'] || '',
+      projectName: project.metadata?.['schema:name'] || '',
```

Comment:
not directly related but when the project metadata didn't get updated to use the compacted form in here, fixed it in a few other places as well cc/ @mhagel